src/explorer.ts

```diff
@@ -20,13 +20,28 @@ export function createKubernetesResource(kind: kuberesources.ResourceKind, id: s
 }
 
 function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
+    if (status === "Running" || status === "Completed") {
         return vscode.Uri.file(path.join(__dirname, "../../images/runningPod.svg"));
     } else {
         return vscode.Uri.file(path.join(__dirname, "../../images/errorPod.svg"));
     }
 }
 
+function parsePodLine(podInfo: string[]): any[] {
```

Comment:
I think we can refine the return type a little, to `[key: string]: string` (which guarantees that it's a map from strings to strings, rather than involving arbitrary value data).

Reply:
perfect, will do that.